### PR TITLE
Expose XS computation

### PR DESCRIPTION
**Pull request by AleCandido**
Expose the coefficients computation, such that it can be used even outside yadism itself.

### CODE REVIEW

File: src/yadism/esf/exs.py [hunk 1]
```diff
@@ -5,7 +5,67 @@
...
+
+def xs_coeffs(kind, y, x=None, Q2=None, params=None):
+    """Compute coefficients in the definition of a given cross section.
+
+    Parameters
+    ----------
+    kind : str
+        the identifier of the cross section for which coefficients have to be
+        computed
+    y : float
```
Comment (felixhekhorn) [inline src/yadism/esf/exs.py:20]: Could you please provide some quick description? (since we're on it)

File: src/yadism/esf/exs.py [hunk 2]
```diff
@@ -5,7 +5,67 @@
...
+    x: None or float
+    Q2: None or float
+    params : None or dict
+
+    """
+    yp = 1.0 + (1.0 - y) ** 2
+    ym = 1.0 - (1.0 - y) ** 2
+    yL = y**2
+
+    # if kind == "XSreduced":
```
Comment (felixhekhorn) [inline src/yadism/esf/exs.py:30]: Let's remove this code which we thought once upon a time we would maybe need - we accept the thing changes dataset by dataset

**Review by felixhekhorn — APPROVED** [review]
Apart from minor things this should be just a reshuffling and so ok ...

File: src/yadism/esf/exs.py (hunk 1 above)
Comment (AleCandido) [reply src/yadism/esf/exs.py:20]: You're right, I'll do it (as you see I started, I simply forgot to complete).

File: src/yadism/esf/exs.py (hunk 2 above)
Comment (AleCandido) [reply src/yadism/esf/exs.py:30]: Ok, I'll do.

Still thinking to improve the names, but for a *long* time being it is just fine.

**Review by felixhekhorn — APPROVED** [review]

**Pull request merged**